src/aiperf/dataset/loader/conflux.py

```
@@ -0,0 +1,1081 @@
+            # paths: explicit agent threading (agent_id + is_subagent + messages)
+            # or the proxy source marker (source == "proxy" + messages).
+            return any(
+                isinstance(r, dict)
+                and "messages" in r
+                and (
+                    ("agent_id" in r and "is_subagent" in r)
+                    or r.get("source") == "proxy"
+                )
+                for r in raw[:20]
+            )
+        except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)